### PR TITLE
Local storage

### PR DESCRIPTION
**Pull request by Alexander-Schulte**
Hey guys here is my next PR! :) 
After you fill out the form, the data is stored in the local Storage with the key "newCourts". If the user submits again a new court, this new object should be added to the array of newCourts. After the user submits the form, the values of the controlled components are set to empty arrays again. 
Thank you for your review! :)

### CODE REVIEW

File: src/components/CourtCards.js [hunk 1]
```diff
@@ -4,13 +4,17 @@ import OpeningIcon from "../images/OpeningIcon.svg";
...
 
 function CourtCards() {
   const [courtName, setCourtName] = useState("");
   const [courtLocation, setCourtLocation] = useState("");
   const [courtOpeningTime, setCourtOpeningTime] = useState("");
   const [courtDescription, setCourtDescription] = useState("");
+  const [courtRating, setCourtRating] = useState("");
 
   function handleCourtName(event) {
     const { value } = event.target;
```
Comment (FriederikeM) [inline src/components/CourtCards.js:20]: i like the deconstruction, would not have thought of that

File: src/components/CourtCards.js [hunk 2]
```diff
@@ -32,16 +36,26 @@ function CourtCards() {
     setCourtDescription(value);
   }
 
+  function handleCourtRating(event) {
+    const { value } = event.target;
+    setCourtRating(value);
```
Comment (FriederikeM) [inline src/components/CourtCards.js:41]: these two functions seem to be doing the same. Maybe you can only define one and then trigger it in both places? (Can't see right now where you are triggering the handleCourtName)
```
function handleInputChange (event) {
const { value } = event.target
setCourtDescription(value);
setCourtRating(value);
}
```
like so?

**Review by FriederikeM — APPROVED** [review]
Looking good, just left a comment for two functions that I think could be reduced to one

**Review by soerendolling — APPROVED** [review]
Nice one implementing local storage. When you open vercel and the form. I can see your stars are not displayed in the drop down menu but I assume that is still under construction

**Pull request merged**